backend/MoleculeOSSite/Models/Validators/LoginValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentValidation;
+using MoleculeOSSite.ModelsDTO;
+
+namespace MoleculeOSSite.Models.Validators
+{
+    public class LoginValidator : AbstractValidator<LoginDTO>
+    {
+        public LoginValidator()
+        {
+            RuleFor(x=>x.UsernameOrEmail).NotEmpty();
+            RuleFor(x=>x.Password).NotEmpty();
+        }
```

Comment:
```suggestion
        {
            RuleFor(x => x.Login).NotEmpty();
            RuleFor(x => x.Password).NotEmpty();
        }
```
1. Odstępy
2. Wystarczy Login a nie jakieś "UsernameOrEmail".